blogs/2018/09/07/blog-post.md

```diff
@@ -0,0 +1,72 @@
+
+## Adopting Azure Pipelines
+
+Earlier this year, we were reached out by the Azure Pipelines (then Visual Studio Team Services) team to try out something new. This was the outcome:
+
+<br />
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">🎉 Hey, did you know <a href="https://twitter.com/code?ref_src=twsrc%5Etfw">@code</a> now uses shiny new <a href="https://twitter.com/VSTS?ref_src=twsrc%5Etfw">@vsts</a> Public Projects and YAML builds to keep its quality up across all 3 platforms simultaneously?! 🤯 Check out the builds at <a href="https://t.co/Q8kYAB1lKG">https://t.co/Q8kYAB1lKG</a>. More about Public Projects at <a href="https://t.co/o0WoNKA4g1">https://t.co/o0WoNKA4g1</a> cc <a href="https://twitter.com/chrisrpatterson?ref_src=twsrc%5Etfw">@chrisrpatterson</a> <a href="https://t.co/Yyhwg19d9z">pic.twitter.com/Yyhwg19d9z</a></p>&mdash; João Moreno (@joaomoreno) <a href="https://twitter.com/joaomoreno/status/991958010661670912?ref_src=twsrc%5Etfw">May 3, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<br />
+
+This announcement marks our move to a more streamlined continuous integration solution. Our builds now run simultaneously across all platforms, [check it out](https://aka.ms/vscode-builds):
+
+![VS Code Build Page](build.png)
+
+There's a lot of cool stuff which needed to happen in order for us to make sure a move. Let's break it down (TODO@PJ: Note for PJ to check on branding – if we should change all these to new naming or not.):
+
+1. [VSTS Public Projects (TODO@PJ: @Dmitry: Do public projects accrue to Azure Pipelines, Azure Repos, or Azure DevOps?)](https://blogs.msdn.microsoft.com/devops/2018/04/27/vsts-public-projects-limited-preview/) enables us to run a [public-facing Visual Studio Code project](https://vscode.visualstudio.com/VSCode/_build?definitionId=1) in which all our continuous integration builds run;
+2. [VSTS Build Agents (TODO@PJ: @Dmitry: Do agents accrue to Azure Pipelines, or Azure DevOps?)](https://docs.microsoft.com/en-us/vsts/pipelines/agents/agents?view=vsts) have long supported the Windows, macOS, & Linux platform matrix;
```

Comment:
Might be DevOps https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents